erpnext/shopping_cart/cart.py

```diff
@@ -96,7 +96,7 @@ def place_order():
 def request_for_quotation():
 	quotation = _get_cart_quotation()
 	quotation.flags.ignore_permissions = True
-	quotation.submit()
+	quotation.save() if get_shopping_cart_settings().draft_quotation else quotation.submit()
```

Comment:
Break this into two lines. Save the quotation. If draft is not checked then submit as well. 
For eg:

```suggestion
	quotation.save() 
	if not get_shopping_cart_settings().draft_quotation: 
	    quotation.submit()
```